lib/util/text/env_vars.go

```diff
@@ -20,6 +22,12 @@ func ContainsEnvVariables(inBytes []byte) bool {
 	return envRegex.Find(inBytes) != nil || escapedEnvRegex.Find(inBytes) != nil
 }
 
+// TargetVarIsJSON returns true if inBytes contains either dots or square brackets
+// signalling that the targetVar is a json object
+func TargetVarIsJSON(inBytes []byte) bool {
+	return jsonPathRegex.Find(inBytes) != nil
+}
```

Comment:
This treats all env variables that have a `.` in it as JSON variables. This should not be causing problems as `.`s in environment variables are not valid notation normally.